generators/client/templates/react/src/main/webapp/app/shared/error-boundary/error-boundary.tsx.ejs

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+class ErrorBoundary extends React.Component<{ children: any }, { error: any; errorInfo: any }> {
+  constructor(props) {
```

Comment:
once type is extracted constructor can be avoided, use
`readonly state: IErrorBoundaryState = { error: undefined, errorInfo: undefined };`